src/plugins/intel_cpu/src/nodes/scatter_update.cpp

```diff
@@ -56,47 +56,147 @@ ScatterUpdate::ScatterUpdate(const std::shared_ptr<ov::Node>& op, const GraphCon
     } else {
         OPENVINO_THROW_NOT_IMPLEMENTED(errorMessage);
     }
-
-    const auto node = ov::as_type_ptr<const ov::op::v12::ScatterElementsUpdate>(op);
-    if (node) {
-        reduction_type = node->get_reduction();
-        use_init_val = node->get_use_init_val();
-    } else {
-        reduction_type = ScatterUpdate::Reduction::NONE;
-    }
-}
-
-void ScatterUpdate::getSupportedDescriptors() {
-    if ((getParentEdges().size() != 3) && (getParentEdges().size() != 4))
-        OPENVINO_THROW(errorPrefix, " has incorrect number of input edges");
-    if (getChildEdges().empty())
-        OPENVINO_THROW(errorPrefix, " has incorrect number of output edges");
-
-    if (getInputShapeAtPort(DATA_ID).getRank() < 1 ||
-        getInputShapeAtPort(INDICES_ID).getRank() < 1 ||
-            getInputShapeAtPort(UPDATE_ID).getRank() < 1) {
-        OPENVINO_THROW(errorPrefix, " do not support scalar input");
-    }
-
+    // In ov::PartialShape with rank 0 (scalars) is converted to ov::intel_cpu::Shape with rank 1.
```

Comment:
By design all the checks below have to be part of `isSupportedOperation` function. It gives an ability to reuse `isSupportedOperation` static function as callback in transformation pipeline to adjust opset lowering (or ops decomposition) in case there is no native support on the plugin level.

Reply:
Added checks for reduction modes to isSupportedOperation. As for scalar, as mentioned in previous comment, will review existing core checks in separate ticket and if possible, remove them from CPU